### PR TITLE
Closed Group Fixes

### PR DESCRIPTION
**Pull request by vincentbavitz**
# Fixed

### Give Feedback when Max Linked Reached

Let the master device know if they're trying to link more than one device.

### Let Linking Device know about linking delay

Let the linking device know that linking may take up to a minute to register on their master device.

### Allow User to Cancel Linking from Registration
image

### Previous Links in Conversations

After attempting many links to `A1` from many different instances, `A2` sees all the pubkeys of the attempted linked devices after it finally links.
Steps to reproduce:
1. Generate new master account
2. Start new link from desktop
3. Cancel linking from slave
4. Repeat 2-3 several times
5. Successfully link a device `A2` to master device `A1`
6. `A2` now sees all the pubkeys of the previously attempted links
Screenshot_20200706_163915

### CODE REVIEW

File: ts/session/protocols/SessionProtocol.ts [hunk 1]
```diff
@@ -105,6 +105,8 @@ export class SessionProtocol {
   public static async sendSessionRequestIfNeeded(
     pubkey: PubKey
   ): Promise<void> {
+    const { ConversationController } = window;
```
Comment (msgmaxim) [inline ts/session/protocols/SessionProtocol.ts:108]: Is this one from Audric's PR?

Comment (vincentbavitz) [reply ts/session/protocols/SessionProtocol.ts:108]: Oop. Yeah, will remove.

File: _locales/en/messages.json [hunk 2]
```diff
@@ -222,7 +222,7 @@
     "description": "Message shown if the import went wrong; first paragraph"
   },
   "importErrorSecond": {
-    "message": "If these steps don't work for you, please submit a debug log (View -> Debug Log) so that we can help you get migrated!",
+    "message": "If these steps don't work for you, please submit a debug log (View 🠖 Debug Log) so that we can help you get migrated!",
```
Comment (Bilb) [inline _locales/en/messages.json:225]: think an error happened here

File: _locales/en/messages.json [hunk 3]
```diff
@@ -862,11 +862,17 @@
...
+  },
+  "devicePairingRequestReceivedLimitDescription": {
+    "message": "To change your linked devices, please unlink a device first."
+  },
   "devicePairingRequestReceivedNoListenerTitle": {
     "message": "Device linking request received."
   },
   "devicePairingRequestReceivedNoListenerDescription": {
-    "message": "Device linking request received but you are not on the device linking screen. \nFirst go to Settings -> Device -> Link New Device."
+    "message": "Device linking request received but you are not on the device linking screen. \nGo to Settings 🠖 Device 🠖 Link New Device."
```
Comment (Bilb) [inline _locales/en/messages.json:875]: same

**Pull request merged**